torchrl/objectives/utils.py

```diff
@@ -29,6 +30,14 @@
     "run `loss_module.make_value_estimator(ValueEstimators.<value_fun>, gamma=val)`."
 )
 
+RANDOM_MODULE_LIST = (
+    nn.Dropout,
+    nn.Dropout2d,
+    nn.Dropout3d,
```

Comment:
Do these guys have a parent, common class?

Reply:
Indeed, all the Dropouts have a common parent _DropoutNd. Ill update it!